idl/github.com/uber/cadence/shared.thrift

```diff
@@ -895,9 +910,7 @@ struct DomainConfiguration {
   10: optional i32 workflowExecutionRetentionPeriodInDays
   20: optional bool emitMetric
   30: optional string archivalBucketName
-  40: optional i32 archivalRetentionPeriodInDays
```

Comment:
why these 2 are deleted?

@andrewjdawson2016

Reply:
synced with Andrew, this is intentional, and since there is no one using it, this should be safe